packages/docusaurus-theme-classic/src/theme/DocItem/index.tsx

```diff
@@ -54,7 +58,9 @@ function DocItem(props: Props): JSX.Element {
   // See https://github.com/facebook/docusaurus/issues/3362
   const showVersionBadge = versions.length > 1;
 
-  const metaTitle = title ? `${title} | ${siteTitle}` : siteTitle;
+  const metaTitle = title
+    ? `${title} ${titleDelimiter || '|'} ${siteTitle}`
```

Comment:
We set the default value for delimeter param via Joi, right? Why do we need to do the same in this component then?

Reply:
I think that you're right and fallback is not required, but I'm not sure if after removal `.empty('')` need be added to the Joi validator (https://joi.dev/api/?v=17.2.1#string).